addon/content/scripts/page-action-panel.js

```diff
@@ -110,9 +110,9 @@ class PageActionPanel {
         timeUnit += "s";
       }
     } else {
-      timeSaved += `${Math.round(timeSeconds)}`;
+      timeSaved += timeSeconds;
       timeUnit += "second";
-      if (Math.round(timeSeconds) !== 1) {
+      if (timeSaved !== 1) {
```

Comment:
So far everything is looking 💯, but I am seeing a possibly new issue with "1 seconds" being pluralized.

![1-seconds-saved](https://user-images.githubusercontent.com/557895/36505849-a1579486-1709-11e8-9179-7d8be760828c.png)

I'm making a wild guess that `timeSaved` is a string (per L102), and then we're concatenating a string on L107. But when we do a plural check on L115, we're checking if the value is `1`. Previously we were using `Math.round()` which was probably auto-casting it to a number, which I don't think is happening any longer.